CHANGELOG.md

```diff
@@ -20,13 +20,53 @@ Keep the changelog pleasant to read in the text editor:
 
 version 2.0.0
 ---------------------------
+
++ Runtime section no longer accepts arbitrary keys. Added new hints section for arbitrary runtime hints.
```

Comment:
Nitpick: The changelog style guide mentions 80 characters per line max. But I am biased since I wrote that style guide :wink:. This is up for debate of course.

Reply:
I think this is the same wrap-vs-no-wrap argument. We should decide on one or the other and be consistent.